apps/galaxy/src/modules/auth/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { useForm } from "react-hook-form";
+import { ErrorMessage } from "@hookform/error-message";
+import useSignIn from "../../../services/signin/useSignIn";
+import { Button, Container, FormLabel, FormControl, Input } from "@cc/ui-chakra";
+
+interface FormData {
+  email: string;
+  password: string;
+}
+
+const SignInForm = () => {
+  const { isLoading, handleSignInSubmit } = useSignIn();
+  const {
+    register,
+    formState: { errors },
+    handleSubmit,
+  } = useForm<FormData>();
+  const formSubmit = ({ email, password }: FormData) => {
```

Comment:
I feels like we can move the `handleSubmit` from line:27 into this function as well, so there will be one place to handle form submission-related logic.

e.g.

```ts
const formSubmit = handleSubmit(handleSignInSubmit);
})

// then in where you defined handleSignInSubmit, it will just take an object that has {email, password}
const handleSignInSubmit = ({email, password}) => {
  // your api logic here...
}
```